recipes-support/rauc-hawkbit-updater/rauc-hawkbit-updater_%.bbappend

```diff
@@ -0,0 +1,14 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
```

Comment:
This is redundant and can be removed. There are currently no files to be prepended here.